client/src/client.ts

```diff
@@ -0,0 +1,29 @@
+import {
+  CancellationToken,
+  LanguageClient,
+  LanguageClientOptions,
+  LSPErrorCodes,
+  RequestType,
+  ServerOptions,
+} from 'vscode-languageclient/node';
+import { log, sleep } from './util';
+
+let client: LanguageClient | null;
+
+export const createClient = (
```

Comment:
This is just a singleton instance of the client that makes it easily accessible.